bin/akula.rs

```diff
@@ -479,14 +535,94 @@ fn main() -> anyhow::Result<()> {
                     },
                     !opt.prune,
                 );
+
                 staged_sync.push(
                     CallTraceIndex {
                         temp_dir: etl_temp_dir.clone(),
                         flush_interval: 50_000,
                     },
                     !opt.prune,
                 );
-                staged_sync.push(Finish, !opt.prune);
+
+                if can_mine {
+                    staged_sync.is_mining = true;
+                    let config = MiningConfig {
+                        enabled: true,
+                        ether_base: opt.mine_etherbase.unwrap().clone(),
```

Comment:
please handle the option gracefully, do not just call unwrap, it will panic when you input None.

Reply:
by the way, codes have many other unwrap calls, please handle them gracefully.